tests/installation/releasenotes.pm

```diff
@@ -23,7 +23,8 @@ sub run() {
     }
 
     # workaround for bsc#1014178
-    if (check_screen('zfcp-popup'), 0) {
+    wait_still_screen(5);
+    if (check_screen('zfcp-popup')) {
```

Comment:
problem is actually it's not `check_screen(…, 0)` but `check_screen(…), 0` which is nonsense. Please see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2253 first